Agda.cabal

```diff
@@ -266,12 +277,12 @@ library
   if impl(ghc < 8.4)
     build-depends: transformers == 0.5.2.0
 
-  -- ASR (2018-10-16). Required for supporting GHC 8.4.1, 8.4.2 and
+  -- ASR (2018-10-16).
+  -- text-1.2.3.0 required for supporting GHC 8.4.1, 8.4.2 and
   -- 8.4.3. See Issue #3277.
-  if impl(ghc >= 8.4.1) && impl(ghc <= 8.4.3)
-    build-depends: text >= 1.2.3.0 && < 1.3
-  else
-    build-depends: text >= 1.2.3.1 && < 1.3
+  -- The other GHC versions can be restricted to >= 1.2.3.1.
+  if impl(ghc < 8.4.1) || impl(ghc > 8.4.3)
+    build-depends: text >= 1.2.3.1
```

Comment:
From the current  `if-then-else` just keep the `if` part. Otherwise we couldn't remove the conditional after dropping support for GHC 8.4.*.

Reply:
Sorry, I do not understand. Together with
```
text >= 1.2.3.0 && < 1.3
```
the current if
```
if impl(ghc >= 8.4.1) && impl(ghc <= 8.4.3)
    build-depends: text >= 1.2.3.0 && < 1.3
```
does not add anything.

What we need to keep from the current `if` is the `else` part, this is why I negated the condition.

Reply:
You are right.